stable/falco/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: falco
-version: 1.0.0
+version: 1.1.0
```

Comment:
Use 1.0.1 instead of 1.1.0

Thank you!

Reply:
Done